packages/Nuget/IPSAG.AbTesting/ResponseModels/ApiResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IPSAG.AbTesting.ResponseModels
+{
+    internal class ApiResponse
```

Comment:
I have removed this response model as it's iuseless.
At the beginning, I'm going to pack the response then I realized that it quite wasted since we have an dedicated ApiErrorResponse to produce error response, then no potential information be added to the 200-OK response except its data.